expyfun/io/_parse.py

```diff
@@ -73,3 +99,102 @@ def read_tab(fname, group_start='trial_id', group_end='trial_ok'):
             d[key] = [(these_vals[ii], these_times[ii]) for ii in idx]
         data.append(d)
     return data
+
+
+def reconstruct_tracker(fname):
+    """Reconstruct TrackerUD and TrackerBinom objects from .tab files. 
+
+    Parameters
+    ----------
+    fname : str
+        Input filename.
+
+    Returns
+    -------
+    tr : list of TrackerUD or TrackerBinom
+        The tracker objects with all responses such that they are in their
+        stopped state (as long as the trackers were allowed to stop during
+        the generation of the file.)
+    """
+    from ..stimuli import TrackerUD, TrackerBinom
+    # read in raw data
+    raw = read_tab_raw(fname)
+
+    # find tracker_identify and make list of IDs
+    tracker_idx = np.where([r[1] == 'tracker_identify' for r in raw])[0]
+    tr = []
```

Comment:
`tracker[0].plot` is fine, just be clear that a list is always returned (it can be length 0, 1, or more than 1 and the user should know to check)